src/main/java/be/uantwerpen/scicraft/renderer/ElectronBlockEntityRenderer.java

```diff
@@ -1,64 +1,33 @@
 package be.uantwerpen.scicraft.renderer;
 
-import be.uantwerpen.scicraft.block.Blocks;
-import be.uantwerpen.scicraft.block.ElectronBlock;
-import be.uantwerpen.scicraft.block.entity.ElectronBlockEntity;
+import be.uantwerpen.scicraft.block.entity.ParticleBlockEntity;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import net.minecraft.client.MinecraftClient;
-import net.minecraft.client.render.RenderLayer;
 import net.minecraft.client.render.VertexConsumerProvider;
 import net.minecraft.client.render.block.entity.BlockEntityRenderer;
 import net.minecraft.client.render.block.entity.BlockEntityRendererFactory;
+import net.minecraft.client.render.block.entity.BlockEntityRendererFactory.Context;
 import net.minecraft.client.util.math.MatrixStack;
-import net.minecraft.nbt.NbtCompound;
-import net.minecraft.network.Packet;
-import net.minecraft.network.PacketByteBuf;
-import net.minecraft.network.listener.ServerPlayPacketListener;
-import net.minecraft.util.math.Vec3f;
-import net.minecraft.util.profiling.jfr.event.PacketSentEvent;
-
-import java.util.Random;
 
 @Environment(EnvType.CLIENT)
-public class ElectronBlockEntityRenderer<T extends ElectronBlockEntity> implements BlockEntityRenderer<T> {
-    public ElectronBlockEntityRenderer(BlockEntityRendererFactory.Context ctx) {}
-
-    @Override
-    public void render(T blockEntity,
-                       float tickDelta,
-                       MatrixStack matrices,
-                       VertexConsumerProvider vertexConsumers,
-                       int light,
-                       int overlay) {
-        matrices.push();
-        double offset;
-        if (blockEntity.is_moving) {
-            offset = (blockEntity.getWorld().getTime() + tickDelta - blockEntity.time) / 40;
-            matrices.translate(0, 0 + offset, 0);
-            if (Math.abs(offset) > 1) {
-                NbtCompound my_nbt = blockEntity.createNbt();
```

Comment:
gebruik de variablen zelf, niet de nbt data. En maak geen aanpassingen hier, ze gebeuren enkel op de client dan. Doe de logica in bijvoorbeeld de tick methode.